examples/glamorous-jest-react/index.test.js

```diff
@@ -0,0 +1,22 @@
+import {matcher, serializer} from 'jest-glamor-react'
+import React from 'react'
+import * as enzyme from 'enzyme'
+import toJson from 'enzyme-to-json'
```

Comment:
Instead of using `toJson` everywhere, we could actually configure Jest to use the serializer like [this](https://github.com/adriantoine/enzyme-to-json#serializer) :)